### PR TITLE
add react1-week1 todo-list files

### PR DESCRIPTION
**Pull request by polyovik**
react1-week1 homework

### CODE REVIEW

File: react/react1/week1/todo-list/src/components/ToDoList.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  { body: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+];
+
+function ToDoList() {
+  const [todoList, setTodos] = useState(initialTodos);
+
+  const getTodos = () => {
+    return todoList.map((todo) => {
+      return (
+        <div className="todos-container">
```
Comment (annamourasobral) [inline react/react1/week1/todo-list/src/components/ToDoList.jsx:15]: I recommend you create another function component with the code contained in this div. :)

**Review by annamourasobral — CHANGES_REQUESTED** [review]
hey, @polyovik 
good job on this delivery!

File: react/react1/week1/todo-list/src/components/ToDoList.jsx (hunk 1 above)
Comment (annamourasobral) [reply react/react1/week1/todo-list/src/components/ToDoList.jsx:15]: Also, remember each child component has to have a key.

Comment (polyovik) [reply react/react1/week1/todo-list/src/components/ToDoList.jsx:15]: Thanks for your time! I will change it in my week2 homework.

**Pull request merged**